content/en/agent/guide/agent-fips-proxy.md

```diff
@@ -342,10 +342,37 @@ For more information about outbound connections from the Agent, see the [Network
 
 If you're still unsure about your issue, you may reach out to the [Datadog support][6].
 
+## FAQ
+
+**1. Does the Datadog Agent and the Datadog FIPS proxy have to be on same host?**
+
+Yes, the FIPS compliance is not retained if the Datadog FIPS proxy and the Datadog Agent are not on the same host.
+The same applies if the `fips.enabled` option in not set to `true` in `datadog.yaml`.
+
+**2. Are customers responsible for hardening their host?**
+
+Yes, **Host security and hardening are customer responsibilities.**
```

Comment:
```suggestion
Yes, host security and hardening are customer responsibilities.
```
The bold here might make this section a little hard to parse for the reader, and it's a one sentence answer, so I don't think we need the added stress here.